bundles/EcommerceFrameworkBundle/EventListener/IndexUpdateListener.php

```diff
@@ -20,7 +20,10 @@
 use Pimcore\Event\Model\DataObjectEvent;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 
-class IndexUpdateListener implements EventSubscriberInterface
+/**
+ * @internal
```

Comment:
I think we should allow to modify that behavior as not every body whats to have index updates on object save.

Reply:
Reverted